ui/nuxeo-document-permissions/nuxeo-document-acl-table.js

```diff
@@ -122,7 +122,11 @@ import './nuxeo-popup-permission.js';
                         [[entityDisplay(ace.username)]]
                       </span>
                     </div>
-                    <div class="flex-2"><span class="label">[[i18n(ace.permission)]]</span></div>
+                    <div class="flex-2">
+                      <span class="label">
+                        [[i18nWithPrefix('label.security.permission', ace.permission)]]
```

Comment:
With the format method you would be able to call something like: `[[formatUserPermission(ace.permission)]]`